packages/lodestar/test/e2e/keymanager/keymanager.test.ts

```diff
@@ -6,17 +6,17 @@ import tmp from "tmp";
 import {createIBeaconConfig, IBeaconConfig, IChainConfig} from "@chainsafe/lodestar-config";
 import {KeymanagerApi, KeymanagerServer} from "@chainsafe/lodestar-keymanager-server";
 import {chainConfig as chainConfigDef} from "@chainsafe/lodestar-config/default";
-import {HttpClient} from "@chainsafe/lodestar-api/src";
-import {getClient} from "@chainsafe/lodestar-api/src/keymanager/client";
+import {HttpClient} from "@chainsafe/lodestar-api";
+import {getClient} from "../../../../api/src/keymanager/client.js";
```

Comment:
Can't use root exports? Why?

Reply:
Only exports declared in package.json can be used. No more reaching randomly into `src/`.